setup.cfg

```diff
@@ -0,0 +1,74 @@
+[metadata]
+name = bapsflib
+author = Basic Plasma Science Facility
+maintainer = Erik T. Everson
+maintainer_email = eeverson@physics.ucla.edu
+description = A toolkit for handling data collected at BaPSF
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = 3-clause BSD
+license_file = LICENSES/LICENSE.txt
+url = https://github.com/BaPSF/bapsflib
+download_url = https://pypi.org/project/bapsflib/
+project_urls =
+    BaPSF = http://plasma.physics.ucla.edu/
+    Documentation = https://bapsflib.readthedocs.io/en/latest/
+    GitHub = https://github.com/BaPSF/bapsflib
+keywords = bapsf, HDF5, lapd, physics, plasma, plasma physics, science
+classifiers =
+    Intended Audience :: Developers
+    Intended Audience :: Education
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Natural Language :: English
+    Operating System :: MacOS
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Education
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Physics
+    Topic :: Software Development
+    Topic :: Software Development :: Libraries :: Python Modules
+
+[options]
+python_requires = >=3.6
+zip_safe = false
+packages = find:
+include_package_data = True
+setup_requires =
+    # PEP-518 encourages the use of pyproject.toml for defining
+    # build dependencies...they should be defined under build-system.requires
+install_requires =
+    # ought to mirror requirements/install.txt
+    astropy >= 3.1
```

Comment:
If we are going to start incorporating plasmapy, then the install should require plasmapy.

Reply:
I see that you changed them.